pkg/controllers/context/context_test.go

```diff
@@ -79,6 +81,20 @@ func TestContext_IsControllerEnabled(t *testing.T) {
 			controllers:                  []string{"alpha", "bravo", "-charlie"}, // --controllers=alpha,bravo,-charlie
 			expected:                     false,
 		},
+		{
+			name:                         "on by star, not in disabled list",
+			controllerName:               "foxtrot",
+			disabledByDefaultControllers: []string{"delta", "echo"},
+			controllers:                  []string{"*"}, // --controllers=*
+			expected:                     true,
+		},
```

Comment:
You can reposition this case so that it is adjacent to similar test cases, making it easier for other users to read and understand.

Reply:
Should i sort the test cases from empty-off-on?